src/popup/components/AccountSelectOptionsItem.vue

```diff
@@ -70,9 +72,16 @@ export default defineComponent({
         : 0,
     );
 
+    const tokenSymbol = computed(
+      () => (props.account.protocol)
```

Comment:
The `protocol` property is required in the account and it is always set so there is no need to check it.